requirements.txt

```diff
@@ -56,3 +56,4 @@ typing_extensions==4.12.2
 uritemplate==4.1.1
 urllib3==1.26.20
 zope.interface==6.4.post2
+sendgrid==6.11.0
```

Comment:
I updated this so that it uses the exact version; less surprises, and a better practice for production environments.